pict-doc/pict/scribblings/pict.scrbl

```diff
@@ -1266,7 +1268,7 @@ to picts, but others are not).
 @defproc[(pict-convertible? [v any/c]) boolean?]{
 Returns @racket[#t] if @racket[v] supports the conversion protocol
 (by being a struct with the @racket[prop:pict-convertible] property)
-and @racket[#f] otherwise.
+and @racket[#f] otherwise. This function returns true for @racket[pict]s.
```

Comment:
@racket[#t], not "true" and either @tech{pict} (if that works) or @racket[pict?].